src/cmd/test.ts

```diff
@@ -0,0 +1,55 @@
+import { unlinkSync, writeFileSync } from 'fs'
```

Comment:
I notice Dockerfile still uses bin/ci-tests.sh for the test phase, should I expect that will continue or will we port it as well?

I am asking because I would like to insert the KinD CI test in our test phase, but it seems the test scripts rely on our Bash test suite.

Reply:
We can port it, but I first wanted the application to be battle tested, before rewriting ci-tests & dockerfiles.